vagrant/motd

```diff
@@ -0,0 +1,14 @@
+Welcome to the mgmt hacking VM.
+
+Please make sure that your authorized key for GitHub is added to the SSH agent
+on the host machine (see `ssh-add -l`).
+
+You can jump into the mgmt repo with the alias `cdrepo`.
+
+To get started, try:
+
+  cdrepo
```

Comment:
Hah check. That is actually the alias I use on my laptop :fist_right: :fist_left: